src/lib/libscf_solver/hf.cc

```diff
@@ -1529,9 +1529,11 @@ void HF::load_orbitals()
 
     boost::shared_ptr<BasisSet> dual_basis;
     if (basisname != options_.get_str("BASIS")) {
-        boost::shared_ptr<BasisSetParser> parser(new Gaussian94BasisSetParser(old_forced_puream));
-        molecule_->set_basis_all_atoms(basisname, "DUAL_BASIS_SCF");
-        dual_basis = BasisSet::construct(parser, molecule_, "DUAL_BASIS_SCF");
+        //boost::shared_ptr<BasisSetParser> parser(new Gaussian94BasisSetParser(old_forced_puream));
+        //molecule_->set_basis_all_atoms(basisname, "DUAL_BASIS_SCF");
+        //dual_basis = BasisSet::construct(parser, molecule_, "DUAL_BASIS_SCF");
+        dual_basis = BasisSet::pyconstruct_orbital(molecule_,
+        "DUAL_BASIS_SCF", basisname, old_forced_puream);
```

Comment:
Seems to work the same either way. So switched to the DUAL label. btw, you can do the following on two sep. files: ``git diff --color-words --no-index tests/sapt4/output.dat ../tests/sapt4/output.ref``